halyard-config/src/main/java/com/netflix/spinnaker/halyard/errors/v1/HalconfigAlert.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.halyard.errors.v1;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This is what Halyard will return to a client in case of an error fixable by the client. This includes validation
```

Comment:
They sound ok, except they leave out the `Fixable` part. I don't feel too strongly about it though.